conda_smithy/templates/travis.yml.tmpl

```diff
@@ -27,6 +27,9 @@ env:
 {%- endif %}
 {% endblock %}
 
+before install:
+    - brew remove --force $(brew list)
```

Comment:
No, but it seems like a waste to leave that stuff around if we don't need it or use it.